com.mojang.Minecraft.json

```diff
@@ -10,31 +10,18 @@
     "tags": [
         "proprietary"
     ],
-    "sdk-extensions": [
-        "org.freedesktop.Sdk.Extension.openjdk11"
-    ],
-    "add-extensions": {
-        "org.freedesktop.Sdk.Extension.openjdk8": {
-            "directory": "jdk8",
-            "version": "20.08",
-            "no-autodownload": true
-        }
-    },
-    "cleanup-commands": [
-        "mkdir -p /app/jdk8"
-    ],
     "finish-args": [
         "--persist=.minecraft",
         "--socket=x11",
```

Comment:
```suggestion
        "--socket=fallback-x11",
```

Reply:
Since this is experimental, I think it's best to leave just both sockets enabled (not using fallback).
That's the approach that electron apps (like element) and chromium use. They also have experimental, opt in wayland support.

Reply:
Ah, ok.